include/dos_system.h

```diff
@@ -49,6 +52,28 @@ enum FatAttributeFlags : uint8_t { // 7-bit
 	DOS_ATTR_DEVICE    = 0b100'0000,
 };
 
+union FatAttributeFlags {
+	uint8_t data = 0;
+
+	bit_view<0, 1> read_only;
+	bit_view<1, 1> hidden;
+	bit_view<2, 1> system;
+	bit_view<3, 1> volume;
+	bit_view<4, 1> directory;
+	bit_view<5, 1> archive;
+	bit_view<6, 1> device;
+	bit_view<7, 1> unused;
+
+	FatAttributeFlags() : data(0) {}
+	FatAttributeFlags(const uint8_t data) : data(data) {}
```

Comment:
the `FatAttributeFlags` `data` argument is ambiguous with the member.
Suggest naming the arg `_data` (or something slightly different).

Reply:
Again, this is a general remark. Fixed, in many places, in a separate commit.